src/SilbinaryWolf/Components/ComponentService.php

```diff
@@ -15,6 +15,10 @@
 
 class ComponentService
 {
+    public $componentTemplatePaths = [
+        'components'
+    ];
```

Comment:
It would be preferable if this could be a config that is accessed via `Config::inst()->get(__CLASS__, 'component_paths')` unless the SSViewer is known to work without the config system. (ie. could be used with the Slim Framework or similar)

My reasoning is that Injector configs are verbose and easy to get wrong, so if we can use simple configs over Injector variables that's preferable.

Reply:
My problem with `Config::inst()->get(__CLASS__, 'property')` is that it's a globally set variable; you can't have different instances of __CLASS__ having different properties here. If you're sure this won't be an issue, I'll swap this over.

Reply:
I'm fairly certain this wouldn't be an issue, because its coupled to how SSViewer does things. You shouldn't be calling `renderComponent`, only SS template code should be.

Reply:
I mean, I can think of some things where calling render component from userland code would be nifty (I'm thinking of partial rendering of components for ajax or edge side inclusion), but they'd need sufficient consideration that a later refactor would likely be a thing anyway. So will swap if your preference is for `Config`